src/laptop_battery_monitor/scripts/laptop_battery.py

```diff
@@ -249,6 +249,8 @@ def __init__(self):
 
         # Battery info
         self._batt_acpi_path = rospy.get_param('~acpi_path', "/sys/class/power_supply/BAT0")
+        # self._batt_acpi_path = rospy.get_param('~acpi_path', "/home/tul1/Desktop/MOCK_BAT")
```

Comment:
Please remove, this file should not be changed.

Reply:
done!